eng/mgmt/Directory.Build.Mgmt.props

```diff
@@ -1,10 +1,15 @@
 ﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(RepoRoot)\tools\bootstrapTools\bootstrap.targets" />
-  <Import Project="CI.Bootstrap.targets" Sdk="Microsoft.Internal.NetSdkBuild.Mgmt.Tools" Version="0.10.0" />
+	<PropertyGroup>
+		<PkgVersion>0.10.0</PkgVersion>
+		<PkgLocalDir>$(RepoRoot)\restoredPackages\microsoft.internal.netsdkbuild.mgmt.tools\$(PkgVersion)\Sdk</PkgLocalDir>
+	</PropertyGroup>
 
-  <PropertyGroup>
-    <SignAssembly>true</SignAssembly>
-    <DelaySign>true</DelaySign>
-    <AssemblyOriginatorKeyFile>$(pkg_root_RepoToolsDir)\MSSharedLibKey.snk</AssemblyOriginatorKeyFile>
-  </PropertyGroup>
+	<Import Condition="Exists($(PkgLocalDir))" Project="$(PkgLocalDir)\CI.Bootstrap.targets" />
```

Comment:
Why did you do this import conditionally here? Just doing the SDK import should work for both cases.